jetcd-core/src/test/java/io/etcd/jetcd/KVNamespaceTest.java

```diff
@@ -0,0 +1,500 @@
+/**
+ * Copyright 2017 The jetcd authors
```

Comment:
Copyright 2019?

Reply:
It has to be "Copyright 2017" since it's a rule listed in checkstyle-checker.xml

Reply:
that's right. we need to update that in the checkstyle-checker.xml. cc/ @lburgazzoli